R/envvars.R

```diff
@@ -70,6 +72,7 @@ c.hipercow_envvars <- function(...) {
   }
   ret <- rlang::inject(rbind(!!!inputs))
   ret <- ret[!duplicated(ret$name, fromLast = TRUE), ]
```

Comment:
Should I have added `drop = FALSE` here as well?

Reply:
I've been doing a bunch with matrices recently where these are really important. I think that for data.frame's mine is superfluous actually - the drop here only would have an effect if we were subsetting columns (in which case with a single column will return the vector and not a data.frame of one column